src/engine/sysDef.cpp

```diff
@@ -675,9 +675,9 @@ const char* DivEngine::getSystemName(DivSystem sys) {
     case DIV_SYSTEM_AMIGA:
       return "Amiga";
     case DIV_SYSTEM_YM2151:
-      return "Yamaha YM2151";
+      return "Yamaha OPM";
```

Comment:
Suggestion: Yamaha YMXXXX (OPX)

Reply:
good idea

Reply:
but then, we have system names vs chip names. doing this will make this option redundant

Reply:
The option will be gone when I re-write the system detection code...

Wait, it won't. NES vs Ricoh 2A03.

Reply:
NES (ricoh 2A03)?